docs/guides/creatingadls.md

```diff
@@ -0,0 +1,140 @@
+# Creating a Bridge With a Dead Letter Sink (DLS)
+
+## What is a Dead Letter Sink?
+
+Knative Eventing provides various configuration parameters to control the delivery of events in case of failure. For instance, you can decide to retry sending events that failed to be consumed, and if this didn't work you can decide to forward those events to a dead letter sink.
+
+## Implementing a Dead Letter Sink
+
+There are two ways to implement a dead letter sink, either by use of a Subscription or by use of a Broker.
+
+### Subscription Configuration
+
+You can configure how events are delivered for each Subscription by adding a delivery spec to the Subscription object, as shown in the following example:
+
+```yaml
+apiVersion: messaging.knative.dev/v1
+kind: Subscription
+metadata:
+  name: example-subscription
+  namespace: example-namespace
+spec:
+  delivery:
+    deadLetterSink:
+      ref:
+        apiVersion: serving.knative.dev/v1
+        kind: Service
+        name: example-sink
+    backoffDelay: <duration>
+    backoffPolicy: <policy-type>
+    retry: <integer>
+```
+
```

Comment:
Might it be relevant to briefly explain the meaning and usage of backoffDelay, backoffPolicy, and retry?